extensions/nodes/nodes.go

```diff
@@ -0,0 +1,114 @@
+package nodes
+
+import (
+	"github.com/rancher/norman/types"
+	v3 "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/kubeapi/secrets"
+	corev1 "k8s.io/api/core/v1"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	initNodeLabelKey         = "rke.cattle.io/init-node"
+	initNodeLabelValue       = "true"
+	local                    = "local"
+	machinePlanSecretType    = "rke.cattle.io/machine-plan"
+	machineNameSteveLabel    = "rke.cattle.io/machine-name"
+	node                     = "node"
+)
+
+// GetInitNode() accepts a client and clusterID and returns the init node object
```

Comment:
nit - can the comment just say `GetInitNode` and not `GetInitNode()`.

Reply:
Done!